agent/04_agent_prepare_release.sh

```diff
@@ -20,9 +20,14 @@ source $SCRIPTDIR/oc_mirror.sh
 # To specify a custom Dockerfile set <ENTRYNAME>_DOCKERFILE, as a relative path of the Dockerfile
 # within the configured repo
 #
-# For example, to use a custom installer image:
+# To specify a custom image name, set <ENTRYNAME>_IMAGE with the required image name
+#
+# For example, to use a custom installer and assisted-service image:
 # export INSTALLER_LOCAL_REPO=~/go/src/github.com/openshift/installer
 # export INSTALLER_DOCKERFILE=images/installer/Dockerfile.ci
+# export ASSISTED_SERVICE_REPO=~/git/assisted-service
```

Comment:
Should this be ?
export ASSISTED_SERVICE**_LOCAL**_REPO=~/git/assisted-service

Reply:
I don't think keeping a `*` in the name portion will work, `REPO_OVERRIDES` looks for names ending with just `_LOCAL_REPO`

Reply:
I think those ** got put in because I tried to make it Bold but I was really asking if 'Local' needs to be added  in 'ASSISTED_SERVICE_REPO'

Reply:
Good catch thanks those asteriks catpured completely my focus :)